src/async-wrap.cc

```diff
@@ -468,6 +468,13 @@ void AsyncWrap::QueueDestroyId(const FunctionCallbackInfo<Value>& args) {
   PushBackDestroyId(Environment::GetCurrent(args), args[0]->NumberValue());
 }
 
+void AsyncWrap::AddWrapMethods(Environment* env,
```

Comment:
Can we bikeshed about the name.
IMHO `Add` -> `Bind` and `Wrap` -> `AsyncHooks` so `BindAsyncHooksMethods`